scalableminds/webknossos
# Handle cell id as vec4 in shader to increase precision

### Mailable description of changes:
 - Fixes an issue with coloring cells in volume tracing (affected hover and made some cells disappear).

### Steps to test:
- I set the active cell to 256 and 512 and labeled some cells. They should have the same color but should not react to the same hover event. 

### Issues:
- fixes #2575 and https://discuss.webknossos.org/t/segment-missing/868/2

------
